pymvr/__init__.py

```diff
@@ -0,0 +1,160 @@
+from typing import List
```

Comment:
Out of curiosity: where did `pymvr` came from? I haven't found it on GitHub.

Reply:
uh, i mentioned pymvr above... my code, but based on pygdtf design. I added it to the DEPENDENCIES.md

Reply:
Very nice! May I suggest releasing it as a python lib on a separate repo, pygdtf really needs it's mvr counterpart on the open source community haha

Reply:
Yes, that is the plan :)